tests/Basics/VelocyPackHelper-test.cpp

```diff
@@ -44,16 +44,23 @@
   r = VPackParser::fromJson(rValue);  \
   EXPECT_EQ(expected, func(l->slice(), r->slice(), true)); \
 
-#define INIT_BUFFER  TRI_string_buffer_t* sb = TRI_CreateStringBuffer();
-#define FREE_BUFFER  TRI_FreeStringBuffer(sb);
-#define STRINGIFY    TRI_StringifyJson(sb, json);
-#define STRING_VALUE sb->_buffer
-#define FREE_JSON    TRI_FreeJson(json);
-
 // -----------------------------------------------------------------------------
 // --SECTION--                                                        test suite
 // -----------------------------------------------------------------------------
 
+TEST(VPackHelperTest, tst_patch_double) {
+  VPackBuilder b;
+  b.add(VPackValue(double(1.0)));
+  
+  EXPECT_DOUBLE_EQ(double(1.0), b.slice().getDouble());
+
+  arangodb::basics::VelocyPackHelper::patchDouble(b.slice(), double(2.0));
+  EXPECT_DOUBLE_EQ(double(2.0), b.slice().getDouble());
+  
+  arangodb::basics::VelocyPackHelper::patchDouble(b.slice(), double(-34.456));
+  EXPECT_DOUBLE_EQ(double(-34.456), b.slice().getDouble());
```

Comment:
`-34.456` can not be converted into a double without loss of precision. As at least the x86-ish CPUs use extended double precision, I suppose this could cause spurious test failures at least with some of the `-ffast-math` optimizations. I'm not sure it'll really happen, but maybe use 
```suggestion
  arangodb::basics::VelocyPackHelper::patchDouble(b.slice(), double(-34.625));
  EXPECT_DOUBLE_EQ(double(-34.625), b.slice().getDouble());
```
or so instead?

Reply:
That's why I used `EXPECT_DOUBLE_EQ` and not `EXPECT_EQ`.